src/main/java/org/fundacionjala/coding/abel/movies/Customer.java

```diff
@@ -0,0 +1,91 @@
+package org.fundacionjala.coding.abel.movies;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by Unknown on 5/26/2017.
+ */
+public class Customer {
+
+    private String name;
+    private List<Rental> rentals;
+    private double totalAmount;
+    private int frequentRenterPoints;
```

Comment:
We don't need totalAmount and totalAmount since we methods to calculate these values